app/Services/RatingsService.php

```diff
@@ -4,13 +4,12 @@
 
 class RatingsService {
 
-  public function updateRatings($request)
+  public function updateRatings($user, $request)
```

Comment:
See my [comment](https://github.com/oneworldcoders/rate-my-church/pull/11#discussion_r450941532) from the other pull request.